include/bluetooth/gatt.h

```diff
@@ -995,6 +999,29 @@ struct bt_gatt_write_params {
  */
 int bt_gatt_write(struct bt_conn *conn, struct bt_gatt_write_params *params);
 
+/** @brief Write Attribute Value by handle without response with callback.
+ *
+ * This function works in the same way as @ref bt_gatt_write_without_response.
+ * With the addition that after sending the write the callback function will be
+ * called.
+ *
+ * Note: By using a callback it also disable the internal flow control
+ * which would prevent sending multiple commands without wainting their
+ * transmissions to complete, so if that is required the caller shall not
+ * submit more data until the callback is called.
+ *
+ * @param conn Connection object.
+ * @param handle Attribute handle.
+ * @param data Data to be written.
+ * @param length Data length.
+ * @param sign Whether to sign data
```

Comment:
The 'func' parameter seems to be undocumented here

Reply:
Will fix it.